app/controllers/ProductController.scala

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import models.Product
+import play.api.Environment
+import play.api.libs.json.Json
+import play.api.mvc.{AbstractController, ControllerComponents}
+
+import java.time.LocalDateTime
+import javax.inject.Inject
+
+// Controller class for whenever a user enters the URL for the bakery.
+class ProductController @Inject() (
```

Comment:
for class comments, use `/** */` scala comments

Reply:
Alright, I will make sure to use this syntax for classes!